src/style.css

```diff
@@ -1 +1,64 @@
 /* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", Arial, sans-serif;
+  --main-color: #00acdc;
```

Comment:
I suggest to define variables in `:root` pseudo selector

Reply:
Hi. Ok, thanks I fixed everything)